src/wp-admin/includes/template.php

```diff
@@ -2432,9 +2438,9 @@ function compression_test() {
  *                                       These key/value attribute pairs will be output as attribute="value",
  *                                       where attribute is the key. Other attributes can also be provided
  *                                       as a string such as 'tabindex="1"', though the array format is
- *                                       preferred. Default null.
+ *                                       preferred. Default empty.
```

Comment:
```suggestion
 *                                       preferred. Default empty string.
```

Be more specific as `empty` can be other values and types.